docs/src/tutorials/conic/tips_and_tricks.jl

```diff
@@ -232,7 +232,7 @@ set_silent(model)
 @objective(model, Max, -t)
 @constraint(model, sum(x) == 1)
 @constraint(model, A * x .<= b)
-@constraint(model, [t; ones(n); x] in MOI.RelativeEntropyCone(2n + 1))
+@constraint(model, (t, ones(n), x) in MOI.RelativeEntropyCone(2n + 1))
```

Comment:
@blegat here's a case where we wouldn't be able to convert from the vector form to the tuple without the dimension of `VectorShape` provided on input because there are two `VectorShape` elements.